docs/content/preview/yugabyte-platform/prepare/cloud-permissions/cloud-permissions-nodes.md

```diff
@@ -56,8 +52,6 @@ Because you are creating the VMs manually, nodes for on-premises providers don't
 
 With an on-premises provider, permissions against your infrastructure are generally not needed to deploy VMs, modify VMs, and so on.
 
-However, lacking these permissions, although YBA can still operate, YBA can't automate many management tasks and will rely on human manual steps or externally-scripted automation to perform the following tasks:
+Provisioning VMs requires root accress, but after VMs have been provisioned with the operating system, required software, and node agent, root and sudo access is no longer required.
 
-- create, scale, and delete database clusters with full automation
-- apply OS security patches
-- recover to full health if a VM fails
+For more information, refer to [Automatically provision on-premises nodes](../../server-nodes-software/software-on-prem/).
```

Comment:
Is there any reasoning behind why we're rewriting this? It doesn't translate to the same meaning as what we are replacing it with.